src/orchard/java.clj

```diff
@@ -363,7 +363,9 @@
       (some (let [classname (.replaceAll path "/" ".")]
               (fn [[prefix url]]
                 (when (.startsWith classname prefix)
-                  (str url path))))
+                  (str (cond-> url
```

Comment:
Might be a good idea to add some comments explaining the need for the special handling.